<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>summaryForm</class>
 <widget class="QWidget" name="summaryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="m_labelTitle">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>New Project Summary</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,97">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="m_labelNamePic">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelName">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>A new </string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,97">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="m_labelSourcesPic">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelSources">
       <property name="text">
        <string>balabala</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,97">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="m_labelConstraintsPic">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelConstraints">
       <property name="text">
        <string>bala  bala</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,97">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="m_labelSDevPic">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelSeriesDevice">
       <property name="text">
        <string>The default series and device for the new project:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>25</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="m_labelSeries">
       <property name="text">
        <string>Series:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelFamily">
       <property name="text">
        <string>Family:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labePackage">
       <property name="text">
        <string>Package:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelDevice">
       <property name="text">
        <string>Device:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>308</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="m_labelTail">
     <property name="text">
      <string>To create the project, click Finish.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
